README.md

```diff
@@ -127,6 +116,18 @@ See our [contribution guidance in the wiki](https://github.com/Microsoft/WinObjC
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+## Advanced Installation for Contributors and Ninjas
```

Comment:
I had fun.

---
In reply to: [107240965](https://github.com/Microsoft/WinObjC/pull/2283#discussion_r107240965) [](ancestors = 107240965)